event_mail/models/event.py

```diff
@@ -16,28 +16,28 @@ def _default_event_mail_template_id(self):
         default=_default_event_mail_template_id,
     )
 
-    @api.onchange("event_mail_template_id")
-    def _onchange_event_mail_template_id(self):
-        vals = [(6, 0, [])]
-        if self.event_mail_template_id.exists():
-            for scheduler in self.event_mail_template_id.scheduler_template_ids:
-                vals.append(
+    @api.depends("event_mail_template_id")
+    def _compute_event_mail_ids(self):
```

Comment:
Nitpicking here but you could do this to split the water, it's slightly cleaner:

```py
records = self.filtered("event_mail_template_id")
without_template = self - records
```

```